cmd/aks-periscope/aks-periscope.go

```diff
@@ -22,25 +23,33 @@ func main() {
 		log.Printf("Failed to create CRD: %+v", err)
 	}
 
+	clusterType := os.Getenv("CLUSTER_TYPE")
+
 	collectors := []interfaces.Collector{}
 	containerLogsCollector := collector.NewContainerLogsCollector(exporter)
 	collectors = append(collectors, containerLogsCollector)
-	systemLogsCollector := collector.NewSystemLogsCollector(exporter)
-	collectors = append(collectors, systemLogsCollector)
 	networkOutboundCollector := collector.NewNetworkOutboundCollector(5, exporter)
 	collectors = append(collectors, networkOutboundCollector)
-	ipTablesCollector := collector.NewIPTablesCollector(exporter)
-	collectors = append(collectors, ipTablesCollector)
-	nodeLogsCollector := collector.NewNodeLogsCollector(exporter)
-	collectors = append(collectors, nodeLogsCollector)
 	dnsCollector := collector.NewDNSCollector(exporter)
 	collectors = append(collectors, dnsCollector)
 	kubeObjectsCollector := collector.NewKubeObjectsCollector(exporter)
 	collectors = append(collectors, kubeObjectsCollector)
+	systemLogsCollector := collector.NewSystemLogsCollector(exporter)
+	ipTablesCollector := collector.NewIPTablesCollector(exporter)
+	nodeLogsCollector := collector.NewNodeLogsCollector(exporter)
 	kubeletCmdCollector := collector.NewKubeletCmdCollector(exporter)
-	collectors = append(collectors, kubeletCmdCollector)
 	systemPerfCollector := collector.NewSystemPerfCollector(exporter)
-	collectors = append(collectors, systemPerfCollector)
+	helmCollector := collector.NewHelmCollector(exporter)
+
+	if clusterType == "connectedCluster" {
```

Comment:
Is it possible that this could come with different casing? Maybe use `strings.EqualsFold`?